mlrun/platforms/iguazio.py

```diff
@@ -375,6 +375,7 @@ def __init__(
         create=True,
         endpoint=None,
         access_key=None,
+        simulated=False,
```

Comment:
We already have the `_DummyStream` class (you created it) which you can use by specifying the `dummy://` in the `stream_path` in `set_tracking`
Why you need this logic here ?

Reply:
i want to simulate the full v3io thing (w dumps..), w dummy we didn't see some of the errors